config_management/roles/kubernetes-install/tasks/debian.yml

```diff
@@ -7,9 +7,9 @@
     state: present
   register: apt_key_k8s_repo
 
-- name: add kubernetes' apt repository (kubernetes-{{ ansible_distribution_release }})
+- name: add kubernetes' apt repository (kubernetes-{{ ansible_distribution_release }}-unstable)
```

Comment:
I'd like to also testing agains unstable somehow easily, not sure what's the best way exactly.